### DISCUSSION TITLE
音源使用问题

### DISCUSSION THREAD

**Discussion by lyh20110** [1715173329/UnblockNeteaseMusic#486] [Q&A] [answered]
我使用node设置音源为qq后,也设置了系统变量qqcookie,但是还是无法播放，提示AggregateError:All promises were rejected,另外在使用yt-dlp音源时无法播放，是需要设置什么参数，怎么设置？

**Comment by MOldtime** [comment]
我也遇到了 有解决方案吗

**Comment by 1715173329** [comment, marked as answer]
Fixed in 9f7e22e93e87295275c936ef11b63872f5f0a51d
